FireboltNETSDK/FireboltClient.cs

```diff
@@ -170,7 +176,7 @@ public async Task<GetSystemEngineUrlResponse> GetSystemEngineUrl(string accountN
         urlBuilder.Query = queryStr.ToString();
         var url = urlBuilder.Uri.ToString();
 
-        return await SendAsync<string>(HttpMethod.Post, url, query, "text/plain", true, cancellationToken);
+        return await SendAsync<string>(HttpMethod.Post, url, query, _textContentType, true, cancellationToken, true);
```

Comment:
It's not clear what `true` argument represents here (both true's in fact)? Can we either create a variable or use named parameters (if c# supports it) so those are a bit more obvious when going through the code?

Reply:
done